.github/workflows/compare-chromium-versions.yml

```diff
@@ -0,0 +1,37 @@
+name: Compare Chromium versions
+
+on:
+  pull_request:
+    types: [opened, reopened, synchronize]
+
+permissions:
+  pull-requests: write
+
+jobs:
+  compare:
+    runs-on: ubuntu-latest
+    env:
+      GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      PR_NUMBER: ${{ github.event.number }}
+    steps:
+      - run: |
+          shopt -s inherit_errexit
+          set -xeEo pipefail
+
+          chromium_ver() { curl -fsS "https://raw.githubusercontent.com/${GITHUB_REPOSITORY:?}/${1:?}/package.json"|jq -r .config.projects.chrome.tag; }
+
+          head_ver="$(chromium_ver "${GITHUB_HEAD_REF:?}")"
+          base_ver="$(chromium_ver "${GITHUB_BASE_REF:?}")"
+
+          if [[ "${head_ver:?}" == "${base_ver:?}" ]]; then
+              echo "The full Chromium version is the same on the PR branch and target branch"
+          elif [[ "${head_ver%%.*}" == "${base_ver%%.*}" ]]; then
+              echo "The major Chromium version is the same on PR branch and target branch"
```

Comment:
let's output these messages using the workflow messaging syntax. Maybe this way it will be visible in the PR checks area?
```
echo "::error::Someting wrong"
echo "::notice::Everything is fine"
```

Let's also reword the messages to better reflect the error and what's allowed. Something like:
```
Chromium version full match: CI ✅ | Merge ✅
Chromium version major match only: CI ✅ | Merge 🚫
Chromium version mismatch: CI 🚫 | Merge 🚫
```